test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/ContactDetailsEmailAddressControllerSpec.scala

```diff
@@ -100,6 +144,89 @@ class ContactDetailsEmailAddressControllerSpec extends ControllerSpec with Defau
           reset(mockRegistrationConnector)
         }
       }
+
+      "return 303 (OK) for not verified email address " + formAction._1 when {
```

Comment:
Do we need test test invalid email addresses here too?

Reply:
@kiranprakashdigital Can you answer this one please?

Reply:
It is handled as a form error "return 400 (BAD_REQUEST)"